src/cli/commands/account.rs

```diff
@@ -74,13 +74,24 @@ impl RunnableCommand<NativeError> for AccountCommand {
                             .map_err(|err| err.to_string())
                     });
 
-                // Open this url
-                open::that(format!(
-                    "{}/registerDevice?spki={}",
-                    GlobalConfig::from_disk().await?.endpoint,
-                    der_url
-                ))
-                .expect("failed to open browser");
+                // URL encoded DER bytes
+                let spki_b64 =
+                    general_purpose::STANDARD.encode(public_device_key.export_bytes().await?);
+                let spki_b64_url_safe =
+                    url::form_urlencoded::byte_serialize(spki_b64.as_bytes()).collect::<String>();
+                // Construct the proper URL to open
+                let url = global
+                    .get_endpoint()
+                    .join(&format!("register-device/{}", spki_b64_url_safe))
```

Comment:
this is not an endpoint we support

Reply:
oh i see yes theres soemthing in the FE for this

Reply:
https://github.com/banyancomputer/banyan-core/pull/270